app/src/main/java/com/github/fribourgsdp/radio/data/view/UserProfileActivity.kt

```diff
@@ -119,7 +119,10 @@ open class UserProfileActivity : MyAppCompatActivity(), KeepOrDismissPlaylistDia
         user.name = usernameField.text.toString()
         //at this point, the userId should be the firebaseUser.uid
         if(user.isGoogleUser){
-            db.setUser(user.id,user)
+            //remove all n
+            user.onlineCopyAndSave()
+            //val onlineUser = user.onlineCopy()
+            //db.setUser(onlineUser.id,onlineUser)
```

Comment:
You forgot to remove those comments ;)

Reply:
You really have falcon eye, good catch.

Reply:
![Falcon gif](https://media.giphy.com/media/2VfNpigH73VWo/giphy.gif)